content/1_General/Cpp_Command.mdx

```diff
@@ -136,7 +136,7 @@ If you want to code in (neo)vim, you can install WSL and code through WSL bash.
 
     If you previously installed these you may need to update them:
 
-    ```
+    ```bash
```

Comment:
I don't believe we have syntax highlighting for bash (I think). (Keep the change though; we might add it in the future.)

If there is sufficient need for bash syntax highlighting we can add it, though this will increase the bundle size.

Reply:
Makes sense; I would leave it out. The benefit is very minimal.